### ISSUE TITLE
⛔ 🐞 Bug Report: Margin added to widgets behaves like padding

### ISSUE THREAD

**Issue by Michaelgimii** [elementor/elementor#20843]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

Hi elementor team.

As you know, when we add a widget in the page, all widgets have a [blue box](https://prnt.sc/Gr-TYCCcKK65). This box defines the border of the widgets.

But the problem is that if we use two padding and margin options in widgets, according to the widgets box, padding and margin are not applied correctly visually. **I emphasize that they are not displayed well visually in the editor.**

In general, when we use padding and margin in a widget, the result is displayed in the editor in such a way that it looks like we are using padding.

### Steps to reproduce

In the video below, I use both padding and margin. But the result is the same in the editor. Pay attention to the widgets box :

https://user-images.githubusercontent.com/88034902/209480391-772a47ca-475c-430e-81ef-f2315b3840fe.mp4

As you can see in the video above, using margin looks exactly like padding. But in bricks, it works fine. In the video below, pay attention to the image widget box :

https://user-images.githubusercontent.com/88034902/209480460-9a93e4b9-7e49-4304-a7c1-4a36b916db8f.mp4

True, it has no effect on the front end. But in the editor we see something that is wrong...

**See another disaster in the Elementor editor in the video below. Of course, I'm not sure if it has anything to do with the problem mentioned above.**

https://user-images.githubusercontent.com/88034902/209480780-71a54f24-e7c5-44e3-b2ac-a3d850465aa3.mp4

In short, the margin option in widgets seems to have a serious problem in the editor. As you can see in the video above, the padding is applied correctly. But Margin ruins everything. 

### Isolating the problem

- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently using the steps above.

### System Info

-

**Comment by nicholaszein** [comment]
Hello @Michaelgimii! 

Thank you for submitting your issue! 🙏 

Please allow us some time to investigate it, and get back to you with our findings. 🧐 

Best regards

**Comment by nicholaszein** [comment]
Hi @Michaelgimii,

after much deliberation, we decided that this issue couldn't be addressed for now. By default, Elementor applies margin and padding to the widget's `.elementor-widget-container` wrapper, which is a child element of the widget wrapper div. This was necessary in the past when CSS Flexbox wasn't yet possible to implement due to a lack of support from browsers. So we had to compromise in order to achieve the desired effects.

We have plans to change this in the future, but because of backward compatibility, we won't be changing this right now. Thanks for understanding! 🙏 

I'm closing this issue as not planned for the time being.

Kind regards
